apps/site/lib/site_web/templates/event/_event_list.html.eex

```diff
@@ -1,13 +1,21 @@
 <div class="page-section event-listing">
-  <%= content_tag(:h2, @title, extra_classes) %>
   <!-- This list-group stuff is bootstrap -->
   <ul class="list-group list-group-flush">
+    <!-- Previous months button. A JS script sets up a listener for button click -->
+    <li class="list-group-item m-previous-events-button">
```

Comment:
Do we need some logic to hide this whole button if there don't happen to be any previous events to hide/show?